packages/next/server/base-server.ts

```diff
@@ -314,6 +315,7 @@ export default abstract class Server {
     this.setAssetPrefix(assetPrefix)
 
     this.incrementalCache = new IncrementalCache({
```

Comment:
I think **eventually** `incrementalCache` shouldn’t be used in the base server. The extended node server should instead import it and find a way to integrate it.

Reply:
Will keep doing to decouple it in the later PRs : )